eventhubs/client/azure-eventhubs/src/test/java/com/azure/eventhubs/ConnectionStringBuilderTest.java

```diff
@@ -0,0 +1,82 @@
+package com.azure.eventhubs;
```

Comment:
Do these tests encompass the ones in `$\eventhubs\data-plane\azure-eventhubs\src\test\java\com\microsoft\azure\eventhubs\connstrbuilder\ConnStrBuilderTest.java`?

Reply:
Most of them. Since we don't set TransportType and OperationTimeOut in ConnectionStringBuilder class Track 2, I skip the tests related to these two properties.